dnf.spec

```diff
@@ -2,7 +2,7 @@
 %undefine __cmake_in_source_build
 
 # default dependencies
-%global hawkey_version 0.54.4
+%global hawkey_version 0.55.0
```

Comment:
This should probably contain more specific version where the required patches are backported - like 0.55.0-5.

Reply:
This is an internal spec that is not use for downstream builds, but I don't know. @inknos What do you think?

Reply:
TL;DR please, set the version to 0.55.0-5. I'll fix the version in libdnf maint branch

Practically speaking; It does not really make much difference now since the specfile in dist-git is different from upstream and the patches to spec are backported manually.
When there is a conflict, i.e. when two patches are fixing the spec for a different version. In that case, I manually edit the patch and remove the diff against spec that's in the patch file. Easy.

Anyway, having specs following the same version requirements is something we want 110% in the future.

I am doing reviews of the maint branch from time to time to see if patches are backported correctly.
I'll fix the version in _libdnf-0.55.0-maint_ branch to be 0.55.0-5 so, please, set 0.55.0-5 here as well :)

Reply:
I will change it

Reply:
@m-blaha @inknos Changed